users.md

```diff
@@ -1,40 +1,59 @@
 ---
-layout: default
+layout: withmap
 title: Cylc Users
 ---
 
 ## Cylc users
 
-Known users include:
+Known users, organised by continent, include:
 
-  * [National Institute of Water and Atmospheric Research
-    (NIWA)](http://www.niwa.co.nz) - New Zealand
-  * [Met Office](http://www.metoffice.gov.uk) - UK
-  * [Max-Planck-Institut f&uuml;r
-      Meteorologie](http://www.mpimet.mpg.de/mpimet-startseite/) - Germany
-  * [Deutches Klimarechenzentrum (DKRZ)](https://www.dkrz.de/) - Germany
-  * [Bureau of Meteorology](http://www.bom.gov.au/) - Australia
-  * [NRL Marine Meteorology Division](http://www.nrlmry.navy.mil) - USA
-  * [557th Weather Wing](https://www.557weatherwing.af.mil) - USA
-  * [Geophysical Fluid Dynamics Laboratory](http://www.gfdl.noaa.gov/) - USA
-  * [Meteorological Service Singapore (MSS)](http://www.weather.gov.sg/home/) - Singapore
-  * [Phillippine Atmospheric, Geophysical, and Astronomical Services
-    Administration (PAGASA)](http://bagong.pagasa.dost.gov.ph)  - Phillippines
-  * [South African Weather Service (SAWS)](http://www.weathersa.co.za/) - South Africa
-  * [National Centre for Medium Range Weather
-      Forecasting](http://www.ncmrwf.gov.in/) - India
-  * [Korean Meteorological Administration (KMA)](http://web.kma.go.kr/) - Korea
+* Oceania:
```

Comment:
Should we sort these lists alphabetically?

Reply:
Just to check, you mean within the sub-lists? Or do you mean alphabetise the overall list, as an alternative to the overall idea of nesting the list by continent & country? Either way makes sense, & I am happy to change as we see best.

Reply:
Just within the sub-lists, and maybe the continents as well. I liked the separation per continent, gave me a much strong impression of the presence of Cylc worldwide :tada: